### ISSUE TITLE
Cannot get Github imdone to load

### ISSUE THREAD

**Issue by patrickml** [imdone/imdone-atom-github#8]
I have tried everything i can think of but i cannot figure out how to get the github version to load. When i go to the tasks this is all i see

https://www.dropbox.com/s/jc7hi9eifqewui4/Screenshot%202015-12-15%2015.22.09.png?dl=0

**Comment by piascikj** [comment]
It might be having trouble recognizing your github repo.  I'll install a patch with some debugging enabled and ask you to open dev tools.  Let you know when that's in.

**Comment by piascikj** [comment]
I just added some console.log in v0.1.8.  Try it again, but this time open the dev tools with ctrl+shift+i and add the pic here.  Just have to drag it into the comment box.

**Comment by patrickml** [comment]
Where does it save the pic?

**Comment by patrickml** [comment]
oh gotcha the dev tools give me a sec

**Comment by patrickml** [comment]
```
::shadow pseudo-element is deprecated. See https://www.chromestatus.com/features/6750456638341120 for more details.
/Users/patricklewis/.atom/packages/emmet/lib/emmet.coffee:139 Loading Emmet extensions from ~/emmet
/Users/patricklewis/.atom/packages/emmet/lib/emmet.coffee:156 Emmet: no such extension folder: /Users/patricklewis/emmet
/Applications/Atom.app/Contents/Resources/app.asar/src/deserializer-manager.js:43 No deserializer found for Object
index.js:51 Window load time: 3111ms
/Applications/Atom.app/Contents/Resources/app.asar/src/text-editor-component.js:915 /deep/ combinator is deprecated. See https://www.chromestatus.com/features/6750456638341120 for more details.
/Users/patricklewis/.atom/packages/imdone-atom/lib/socket-server.coffee:19 port in use
/Users/patricklewis/.atom/packages/imdone-atom/lib/socket-server.coffee:21 Error: listen EADDRINUSE :::9799(…)
/Users/patricklewis/.atom/packages/imdone-atom-github/lib/github-service.coffee:47 *** Found git repo with origin:git@github.com:baceapp/flex.git ***
/Users/patricklewis/.atom/packages/imdone-atom-github/lib/github-service.coffee:51 *** Found upstream branch: refs/remotes/origin/master with target: ****MyGitHupAuthKey**** ***
```

**Comment by piascikj** [comment]
@patrickml Thanks for reporting this one.  I think I got it fixed up in v0.1.9.  Just published it.

**Comment by patrickml** [comment]
+1 looks like you got it fixed. I got a few questions about the package. Is there anyway to make it when you link an issue with a line that it posts in github the file + linenumber + the TODO its self? I figured it would give great reference to anyone who views the issue later. Also as you check things off it could mark the TODO on github as done showing other people your progress.
